### ISSUE TITLE
Bridges Template missing --gres=gpu:p100:2 with GPU-shared partition

### ISSUE THREAD

**Issue by evanmiller326** [glotzerlab/signac-flow#59]
### Description

When trying to submit a job to the GPU-shared partition on Bridges with Signac Flow version 0.6.4 results in a failure ` sbatch: error: No gpu count given, specify --gres=gpu `. 

### To reproduce

Within the template bridges.sh where the resources are allocated based on the partition chosen:

```
{% elif partition == 'GPU-shared' %}
#SBATCH -N {{ nn|default(1, true)|check_utilization(gpu_tasks, 1, threshold, 'GPU') }}
#SBATCH --ntasks-per-node=16
{% elif 'shared' in partition %}
#SBATCH -N {{ nn|default(1, true) }}
#SBATCH --ntasks-per-node={{ cpu_tasks }}
{% else %}
```
This section needs ` #SBATCH --gres=gpu:p100:2` 

### Error output

If possible, copy any terminal outputs or attach screenshots that provide additional information on the problem.

``` 
sbatch: error: No gpu count given, specify --gres=gpu
             See https://www.psc.edu/bridges/user-guide
sbatch: error: Batch job submission failed: Invalid generic resource (gres) specification
Submission error: sbatch error:
make: *** [run] Error 1 
```

### System configuration

Please complete the following information:

```
Linux-3.10.0-693.21.1.el7.x86_64-x86_64-with-centos-7.4.1708-Core
3.7.1 (default, Dec 14 2018, 19:28:38)
[GCC 7.3.0]
0.9.5
0.6.4
```

**Comment by mikemhenry** [comment]
Looks like a regression from here https://github.com/glotzerlab/signac-flow/commit/5b02ecf8f55453675f47decfbdd5a862090b2bb7#diff-309c6ee2196cea793ac60cc84f55be35L35

Thanks for the report @evanmiller326 :tada:

**Comment by csadorf** [comment]
Fixed with merge of PR #69 .
